.github/workflows/build-deps.yml

```diff
@@ -0,0 +1,196 @@
+          -DLLVM_USE_CRT_RELEASE=MT ^
+          -DLLVM_USE_CRT_DEBUG=MTd 
+        cd build_arm64
+        ninja -v install-distribution
+
+    - name: Pack llvm-arm64.zip
+      run: |
+        cd llvm-project/build_arm64
+        $unneeded_tool = 'clang-check', 'clang-cl', 'clang-cpp', 
+          'clang-extdef-mapping', 'clang-format',
+          'clang-func-mapping', 'clang-import-test',
+          'clang-linker-wrapper', 'clang-offload-bundler',
+          'clang-offload-packager', 'clang-refactor',
+          'clang-rename', 'clang-repl', 'clang-scan-deps',
+          'diagtool', 'git-clang-format', 'hmaptool', 'ld.lld',
+          'ld64.lld', 'ld64.lld.darwinnew', 'ld64.lld.darwinold',
+          'lld-link', 'libclang.dll', 'llvm-cov', 'llvm-ml',
+          'llvm-lib', 'llvm-pdbutil', 'llvm-profdata',
+          'llvm-rc'
+        foreach ($tool in $unneeded_tool) {
+            $tool = "bin" + $tool + ".exe"
+            if (Test-Path $tool) {
+              "Remove " + $tool
+              Remove-Item $tool
+            }
+          }
+        Compress-Archive -Path bin,lib\clang -DestinationPath llvm-arm64
+
+    - name: Archive llvm-arm64.zip
+      uses: actions/upload-artifact@v3
+      with:
+        name: llvm-arm64.zip
+        path: llvm-project/build_arm64/llvm-arm64.zip
+        retention-days: 1
+
+  build-llvm-webassembly-arm64_cross_compilation:
+    if: ${{ inputs.llvm_build_llvm_build_cross_compilation }}
+
+    runs-on: windows-latest
+
+    steps:
+    
+    - name: Build LLVM WebAssembly arm64 (Cross Compilation)
+      run: |
+        Set-PSDebug -Trace 1
+        git clone --config core.autocrlf=false https://github.com/llvm/llvm-project.git --single-branch --branch main --depth 1
+        cd llvm-project
+        cmake -S llvm -B build_host -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD=""
+        cmake --build build_host --target llvm-tblgen llvm-nm clang-tblgen --config MinSizeRel
+        $llvm_root = Get-Location
+        cmake -S llvm -B build_arm64 -A ARM64 `
+          -DLLVM_TABLEGEN="$llvm_root\build_host\MinSizeRel\bin\llvm-tblgen.exe" `
+          -DCLANG_TABLEGEN="$llvm_root\build_host\MinSizeRel\bin\clang-tblgen.exe" `
+          -DLLVM_NM="$llvm_root\build_host\MinSizeRel\bin\llvm-nm.exe" `
+          -DLLVM_ENABLE_PROJECTS='lld;clang' -DLLVM_TARGETS_TO_BUILD="host;WebAssembly" `
+          -DLLVM_INCLUDE_EXAMPLES=OFF -DLLVM_INCLUDE_TESTS=OFF
+        cmake --build build_arm64 --config MinSizeRel
+
+  build-nodejs-arm64:
+    if: ${{ inputs.nodejs_build }}
+
+    runs-on: [self-hosted, Windows, ARM64, WASM]
+
+    steps:
+
+    - name: Build Node.js arm64
+      run: |
+        Set-PSDebug -Trace 1
+        git clone https://github.com/Windows-on-ARM-Experiments/node.git --single-branch -b fix-arm64-compilation --depth 1
+        cd node
+        .\vcbuild openssl-no-asm arm64
```

Comment:
JFYI, since some recend update of the upstream, the `openssl-no-asm` is no longer needed. My PR with Node artifact generation (https://github.com/Windows-on-ARM-Experiments/emsdk/pull/9) will be removing it...